@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -1,10 +1,42 @@
 /** @jsx jsx */
 import PropTypes from 'prop-types'
 import { jsx } from 'theme-ui'
+import typy from 'typy'
 
 const DownloadCitation = ({ iiifManifest }) => {
+  let biblio = ''
+  const metadata = {}
+  if (iiifManifest.type === 'Collection') {
+    biblio = iiifManifest.label.en[0] + ', ' + iiifManifest.slug.replace('item/', '') + '. ' +
+      iiifManifest.provider[0].homepage[0].label.en[0] + ', Hesburgh Libraries, University of Notre Dame, South Bend, IN. ' +
```

Comment:
We really don't want to hard code anything specific to us within the `@ndlib/gatsby-theme-marble` folder, that needs to live in the `site` folder. So we will need to either do this through a shadow component or an i18n translation.